chain-connect/src/GalachainConnectClient.ts

```diff
@@ -26,12 +26,16 @@ export class GalachainConnectClient {
   #provider: BrowserProvider | undefined;
   #chainCodeUrl: string;
 
-  get address() {
+  get ethAddress() {
     return `eth|${this.#address}`;
   }
 
+  get galachainAddress() {
+    return getAddress(this.#address);
```

Comment:
This is attempting to return an eth address, should be `return this.ethAddress.replace('0x', 'eth|')`